### ISSUE TITLE
ServiceExceptionResult not configurable

### ISSUE THREAD

**Issue by elblondo** [Xciles/Uncommon#2]
Hi Xciles,

Would it be possible for you to make the ServiceExceptionResult object replaceable with a custom ExceptionResult object?

**Comment by Xciles** [comment]
The handling of a service exception will be changed to a more generic solution. 
This will mean that the current ServiceExceptionResult will no longer be available....

The basic idea behind it is to no longer parse the json and the uncommonrequestexception will get a method to convert the returned exception json to your T.

**Comment by Xciles** [comment]
As a result of this request I've marked 'ServiceExceptionResult' Obsolete. 

Custom exception results returned by a server have been changed to be converted to raw string (JSON). This result can then be converted to a given type of your choice using 'ConvertExceptionResponseToObject<T>()' on the thrown UncommonRequestException.

Currently the raw string is Internal. 

Commit: f670e95eeb7df8a56d18197ace410b0fa0401549

Change has been already pushed to nuget as beta: 1.3.4-beta.
